packages/web-components/src/components/va-link/va-link.tsx

```diff
@@ -0,0 +1,169 @@
+import { Component, Event, EventEmitter, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'va-link',
+  styleUrl: 'va-link.css',
+  shadow: true,
+})
+export class VaLink {
+  /**
+   * If `true`, the anchor text will be bolded and include a right arrow icon.
+   */
+  @Prop() active?: boolean = false;
+
+  /**
+   * If `true`, a channel icon will be displayed before the anchor text.
+   */
+  @Prop() channel?: boolean = false;
+
+  /**
+   * If `true`, the component-library-analytics event is disabled.
+   */
+  @Prop() disableAnalytics?: boolean = false;
+
+  /**
+   * If `true`, the link will be treated as a download, and a download icon will be displayed before the anchor text.
+   */
+  @Prop() download?: boolean = false;
+
+  /**
+   * The href attribute of the anchor.
+   */
+  @Prop() href!: string;
+
+  /**
+   * The suggested filename. Only valid if download is `true`.
+   */
+  @Prop() filename?: string;
+
+  /**
+   * The type of the file. Only displayed if download is `true`.
+   */
+  @Prop() filetype?: 'PDF';
+
+  /**
+   * The number of pages of the file. Only displayed if download is `true`.
+   */
+  @Prop() pages?: number;
+
+  /**
+   * The anchor text.
+   */
+  @Prop() text!: string;
+
+  /**
+   * If `true`, a video icon will be displayed before the anchor text.
+   */
+  @Prop() video?: boolean = false;
+
+  /**
+   * The event used to track usage of the component.
+   */
+  @Event({
+    bubbles: true,
+    composed: true,
+    eventName: 'component-library-analytics',
+  })
+  componentLibraryAnalytics: EventEmitter;
+
+  private handleClick = (e: MouseEvent): void => {
+    // TODO: remove console.log
+    console.log(e);
```

Comment:
Just highlighting this, though I'm sure you're aware since you left a TODO.

Reply:
This is here to allow the web component to build. I can remove the function parameter or leave the console log until we're using the event in the analytics event.